File: CS/MvcDashboardDataSources/Data/OrderContext.cs
namespace MvcDashboardDataSources
{
    using System.Data.Entity;

    public partial class OrderContext : DbContext
    {
        public OrderContext()
            : base("name=OrderContext")
        {
        }

        public virtual DbSet<OrderDetails> OrderDetails { get; set; }

        protected override void OnModelCreating(DbModelBuilder modelBuilder)
        {
            modelBuilder.Entity<OrderDetails>()
                .Property(e => e.UnitPrice)
                .HasPrecision(10, 4);
        }
    }
}

File: VB/MvcDashboardDataSources/Data/OrderContext.vb
Imports System.Data.Entity

Namespace MvcDashboardDataSources

	Partial Public Class OrderContext
		Inherits DbContext

		Public Sub New()
			MyBase.New("name=OrderContext")
		End Sub

		Public Overridable Property OrderDetails() As DbSet(Of OrderDetails)

		Protected Overrides Sub OnModelCreating(ByVal modelBuilder As DbModelBuilder)
			modelBuilder.Entity(Of OrderDetails)().Property(Function(e) e.UnitPrice).HasPrecision(10, 4)
		End Sub
	End Class
End Namespace
